package.json

```diff
@@ -53,5 +55,10 @@
 			"last 1 firefox version",
 			"last 1 safari version"
 		]
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "pretty-quick --staged && npm build && npm lint"
```

Comment:
Looks great, but we want to run the build, lint and then pretty quick .... because if you think about it, what's the point of formatting all your documents if the build or lint is broken.